checkov/terraform/checks/resource/alicloud/LogAuditRDSEnabled.py

```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class LogAuditRDSEnabled(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure log audit is enabled for RDS"
+        id = "CKV_ALI_38"
+        supported_resources = ['alicloud_log_audit']
+        categories = [CheckCategories.ENCRYPTION]
```

Comment:
```suggestion
        categories = [CheckCategories.LOGGING]
```
probably fits better 😄